axelor-quality/src/main/resources/views/TrackingNumber.xml

```diff
@@ -0,0 +1,70 @@
+      <insert position="after">
+        <field name="conformitySelect"/>
+      </insert>
+    </extend>
+  </grid>
+
+  <form name="tracking-number-form" title="Tracking number" id="quality-tracking-number-form"
+    model="com.axelor.apps.stock.db.TrackingNumber" extension="true">
+    <extend target="//panel-tabs[@name='trackingNumberPanelTabs']">
+      <insert position="inside">
+        <panel name="characteristicsPanel" title="Characteristics" colSpan="12">
+          <field name="conformitySelect"/>
+          <button onClick="save,action-quality-method-tracking-number-add-complete-control"
+            name="AddCompleteControlBtn" title="Add a complete control" colSpan="3"/>
+          <button
+            onClick="save,action-quality-view-tracking-number-show-product-characteristic"
+            name="AddCharacteristicControlBtn" title="Add a characteristic control" colSpan="3"/>
+          <field name="trackingNumberCharacteristicList" colSpan="12" canNew="false"
+            onChange="action-quality-method-tracking-number-assign-conformity-select"/>
+        </panel>
+      </insert>
+    </extend>
+  </form>
+
+  <form model="com.axelor.apps.stock.db.TrackingNumber" title="Product characteristics"
+    name="tracking-number-characteristic-product-characteristic-wizard-form">
+    <panel name="mainPanel">
+      <panel-related name="productCharacteristicPanel"
+        field="product.productCharacteristicSet" grid-view="product-characteristic-grid"
+        form-view="product-characteristic-form" colSpan="12" canNew="false" editable="true"/>
+      <button onClick="action-quality-method-tracking-number-add-characteristic-control"
+        name="AddCharacteristicControlBtn" title="Add a characteristic control"/>
+    </panel>
+  </form>
+
+  <action-method name="action-quality-method-tracking-number-assign-conformity-select">
+    <call class="com.axelor.apps.quality.web.TrackingNumberController"
+      method="assignConformitySelect"/>
+  </action-method>
+
+  <action-method name="action-quality-method-tracking-number-add-complete-control">
+    <call class="com.axelor.apps.quality.web.TrackingNumberController"
+      method="addCompleteControl"/>
+  </action-method>
+
+  <action-view name="action-quality-view-tracking-number-show-product-characteristic"
+    title="Product characteristic" model="com.axelor.apps.stock.db.TrackingNumber">
+    <view type="form" name="tracking-number-characteristic-product-characteristic-wizard-form"/>
+    <view-param name="popup" value="reload"/>
+    <view-param name="popup-save" value="false"/>
+    <view-param name="show-toolbar" value="false"/>
+    <view-param name="show-confirm" value="false"/>
+    <context name="_showRecord" expr="eval: id"/>
+  </action-view>
+
+  <action-method
+    name="action-quality-method-tracking-number-add-characteristic-control">
+    <call class="com.axelor.apps.quality.web.TrackingNumberController"
+      method="addCharacteristicControl"/>
+  </action-method>
```

Comment:
Please review the actions name and apply naming convention:
`action-<objectname>-<actiontype>-<context>`